<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>770</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>770</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>560</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="outputLBL">
       <property name="minimumSize">
        <size>
         <width>120</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>135</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Select a prop for the camera to follow.</string>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel{color: #82be0f;}</string>
       </property>
       <property name="text">
        <string>Output Fbx Folder :</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="fbxpath">
       <property name="minimumSize">
        <size>
         <width>296</width>
         <height>22</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>200</width>
         <height>22</height>
        </size>
       </property>
       <property name="text">
        <string>None</string>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="loadBTN">
       <property name="minimumSize">
        <size>
         <width>28</width>
         <height>25</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>33</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="avatarLBL">
       <property name="minimumSize">
        <size>
         <width>120</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>135</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Select a prop for the camera to follow.</string>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel{color: #82be0f;}</string>
       </property>
       <property name="text">
        <string>Object (Avatar) :</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="control">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>240</width>
         <height>0</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_9">
     <item>
      <widget class="QLabel" name="propLBL">
       <property name="minimumSize">
        <size>
         <width>120</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>135</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Select a prop for the camera to follow.</string>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel{color: #82be0f;}</string>
       </property>
       <property name="text">
        <string>Object (Prop) :</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="prop">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>240</width>
         <height>0</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_6">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="3" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QLabel" name="cameraLBL">
       <property name="minimumSize">
        <size>
         <width>120</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>135</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Select a prop for the camera to follow.</string>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel{color: #82be0f;}</string>
       </property>
       <property name="text">
        <string>Object (Camera) :</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="camera">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>240</width>
         <height>0</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_5">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="4" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <item>
      <widget class="QCheckBox" name="checkAvatar">
       <property name="text">
        <string>Export Avatar</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkProp">
       <property name="text">
        <string>Export Prop</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkCamera">
       <property name="text">
        <string>Export Camera</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="5" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="formatLBL">
       <property name="minimumSize">
        <size>
         <width>120</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>135</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Select a prop for the camera to follow.</string>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel{color: #82be0f;}</string>
       </property>
       <property name="text">
        <string>Texture Format:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="format">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>120</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>120</width>
         <height>16777215</height>
        </size>
       </property>
       <item>
        <property name="text">
         <string>Default</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Bmp</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Jpeg</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Tga</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Png</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Tif</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="6" column="0">
    <widget class="QGroupBox" name="groupBox_4">
     <property name="toolTip">
      <string>Start and end frame for the entire follow animation.</string>
     </property>
     <property name="title">
      <string>Timeline (Frames)</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_8">
      <property name="leftMargin">
       <number>6</number>
      </property>
      <property name="topMargin">
       <number>10</number>
      </property>
      <property name="rightMargin">
       <number>6</number>
      </property>
      <property name="bottomMargin">
       <number>3</number>
      </property>
      <item>
       <widget class="QLabel" name="label_7">
        <property name="minimumSize">
         <size>
          <width>30</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>30</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Start :</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="start_frame">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>18</height>
         </size>
        </property>
        <property name="maximum">
         <number>100000</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_8">
        <property name="minimumSize">
         <size>
          <width>30</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>30</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>End :</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="end_frame">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>18</height>
         </size>
        </property>
        <property name="maximum">
         <number>100000</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="minimumSize">
      <size>
       <width>462</width>
       <height>220</height>
      </size>
     </property>
     <property name="title">
      <string>FBX Export Options</string>
     </property>
     <widget class="QCheckBox" name="AutoSkinRigidMesh">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>151</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Auto Skin Rigid Mesh</string>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QCheckBox" name="ExportRootMotion">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>141</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Export Root Motion</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QCheckBox" name="ZeroMotionRoot">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>131</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Zero Motion Root</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QCheckBox" name="ExportPbrTextureAsImageInFormatDirectory">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>180</y>
        <width>291</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Export PbrTexture AsImage In Format Directory</string>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QCheckBox" name="InverseNormalY">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>131</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Inverse Normal Y</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QCheckBox" name="UnrealEngine4BoneAxis">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>100</y>
        <width>171</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>UnrealEngine4 Bone Axis</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QCheckBox" name="RenameDuplicateBoneName">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>120</y>
        <width>211</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Rename Duplicate Bone Name</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QCheckBox" name="RenameDuplicateMaterialName">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>140</y>
        <width>221</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Rename Duplicate Material Name</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QCheckBox" name="RenameTransparencyWithPostFix">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>160</y>
        <width>231</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Rename Transparency With Post Fix</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QCheckBox" name="UnrealIkBone">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>140</y>
        <width>131</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Unreal Ik Bone</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QCheckBox" name="ResetSelfillumination">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>80</y>
        <width>171</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Reset Self illumination</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QCheckBox" name="ResetBoneScale">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>60</y>
        <width>131</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Reset Bone Scale</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QCheckBox" name="BakeMouthOpenMotionToMesh">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>120</y>
        <width>211</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Bake Mouth Open Motion To Mesh</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QCheckBox" name="RenameBoneToLowerCase">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>40</y>
        <width>201</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Rename Bone To Lower Case</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QCheckBox" name="RenameBoneRootToGameType">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>20</y>
        <width>231</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Rename Bone Root To Game Type</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QCheckBox" name="ExtraWordForUnityAndUnreal">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>100</y>
        <width>211</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Extra Word For Unity And Unreal</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QCheckBox" name="UnrealPreset">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>160</y>
        <width>131</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Unreal Preset</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QCheckBox" name="RemoveAllMesh">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>180</y>
        <width>141</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Remove All Mesh</string>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
     </widget>
    </widget>
   </item>
   <item row="8" column="0">
    <widget class="QLabel" name="message">
     <property name="styleSheet">
      <string notr="true">color: grey;</string>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;1.选择Avatar 在Content|Animation Custom|Motion下创建它的Motion，&lt;/p&gt;&lt;p&gt;并用Avatar的名称命名它;Prop和Camera需要手动导出FBX到UE4的项目目录&lt;/p&gt;&lt;p&gt;Content|icloneFBX下面，prop|camera名称.fbx。&lt;/p&gt;&lt;p&gt;2.在Output Fbx Folder里设置UE4的项目目录 Content|icloneFBX目录，&lt;/p&gt;&lt;p&gt;选择一个场景里的Avatar。&lt;/p&gt;&lt;p&gt;3.在FBX Export Options里面选择要输出的Avatar\Prop\Camera。&lt;/p&gt;&lt;p&gt;4.点击Export按钮，开始输出。完成后在UE4原有的RLContent层级下&lt;/p&gt;&lt;p&gt;的资产动画会被更新。&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="indent">
      <number>12</number>
     </property>
    </widget>
   </item>
   <item row="9" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>10</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="10" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="ue4projectLBL">
       <property name="minimumSize">
        <size>
         <width>90</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>90</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Select a prop for the camera to follow.</string>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel{color: #82be0f;}</string>
       </property>
       <property name="text">
        <string>UE4 Project :</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="uepath">
       <property name="minimumSize">
        <size>
         <width>296</width>
         <height>22</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>200</width>
         <height>22</height>
        </size>
       </property>
       <property name="text">
        <string>None</string>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="loadueBTN">
       <property name="minimumSize">
        <size>
         <width>28</width>
         <height>25</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>33</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="11" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QPushButton" name="updateBTN">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>20</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Resets all of the parameters to their default values.</string>
       </property>
       <property name="text">
        <string>Update</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="exportBTN">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>20</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Apply keyframes for the smooth camera follow.</string>
       </property>
       <property name="text">
        <string>Export</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
